Doc/library/tkinter.rst

```diff
@@ -32,17 +37,28 @@ this should open a window demonstrating a simple Tk interface.
    `Tkinter docs from effbot <http://effbot.org/tkinterbook/>`_
       Online reference for tkinter supported by effbot.org.
 
-   `Tcl/Tk manual <https://www.tcl.tk/man/tcl8.5/>`_
-      Official manual for the latest tcl/tk version.
-
    `Programming Python <http://learning-python.com/about-pp4e.html>`_
       Book by Mark Lutz, has excellent coverage of Tkinter.
 
    `Modern Tkinter for Busy Python Developers <https://www.amazon.com/Modern-Tkinter-Python-Developers-ebook/dp/B0071QDNLO/>`_
       Book by Mark Rozerman about building attractive and modern graphical user interfaces with Python and Tkinter.
 
    `Python and Tkinter Programming <https://www.manning.com/books/python-and-tkinter-programming>`_
-      The book by John Grayson (ISBN 1-884777-81-3).
+      Book by John Grayson (ISBN 1-884777-81-3).
+
+   Tcl/Tk documentation:
+
+   `Tcl/Tk recent man pages <https://www.tcl.tk/doc/>`_
+      Recent Tcl/Tk manuals on www.tcl.tk.
```

Comment:
This is the heart of the issue.  I see two deficiencies wtih the link above.
1.  This link has quick links to 8.6 and 8.7 but not 8.5.  
2. Each link goes to an general tcl/tk index page for 8.x with 16 entries.  One has to know to then click the following, which will not be obvious to everyone.
```
Tk Commands
    The additional commands which the wish interpreter implements.
```
I was initially confused because I did not know until now that there are two tcl shell/interpreters and that tkinter is implementing the wish add-ons, and not core tcl/tk, or perhaps the following.
```
Tk Library
    The additional C functions which a Tk extended C program may use.
```
So I *strongly* suggest adding *at the top* what 99% of users of this section need to read.
```
Tk commands <https://www.tcl.tk/man/tcl8.6/TkCmd/contents.htm>
  Commands without '__' in their name correspond to tkinter and ttk widget classes.
  Change '6' to '4', '5', or '7' for the corresponding tk/tkinter versions.
```

Reply:
I agree. Since we are bundling 8.6 it does make sense to make it the default link, while pointing out how to get to the other versions too.

Reply:
Oh, I missed that @andresdelfino moved one level up. The link to the default Tcl/Tk manual is https://www.tcl.tk/man/tcl/.

I think this link is for more advanced users, and for beginners the link to effbot.org and similar resources is more preferable. Maybe we should copy the content from external resources.

Reply:
If you are saying that our doc *should* include a tk widget reference, I agree.  Another issue.